universal_transfer_operator/src/universal_transfer_operator/datasets/file/types/__init__.py

```diff
@@ -3,24 +3,24 @@
 import pathlib
 
 from universal_transfer_operator.constants import FileType as FileTypeConstants
-from universal_transfer_operator.datasets.file.types.base import FileType
-from universal_transfer_operator.datasets.file.types.csv import CSVFileType
-from universal_transfer_operator.datasets.file.types.json import JSONFileType
-from universal_transfer_operator.datasets.file.types.ndjson import NDJsonFileType
-from universal_transfer_operator.datasets.file.types.parquet import ParquetFileType
+from universal_transfer_operator.datasets.file.types.base import FileTypes
+from universal_transfer_operator.datasets.file.types.csv import CSVFileTypes
```

Comment:
I think there is an enum with `CSVFileType` in constant which is separate and this is class `CSVFileTypes` which inherits from `FileTypes` class.

Reply:
No, my point is that the name should be CSVFileType as it is singular because it is a single type and not a set of types. Maybe as a separate we can do this.